crates/viewer/re_space_view_spatial/src/visualizers/utilities/entity_iterator.rs

```diff
@@ -178,12 +134,11 @@ where
             space_view_class_identifier: view_ctx.space_view_class_identifier(),
         };
 
-        let results = query_archetype_with_history::<A>(
+        let results = data_result.query_archetype_with_history::<A>(
             ctx,
             &query.timeline,
             query.latest_at,
             data_result.query_range(),
```

Comment:
Do we ever pass anything but the data_result's query range in here? If not then we can drop that argument :)

Also if I'm not mistaken, `re_space_view` knows about the `ViewQuery` type we're using here, so we might just pass a reference to that instead.


Disregard that if there's friction, I don't want you to have to expand your tangent here more than necessary.

Reply:
👀 (maybe tomorrow)

Reply:
All done, very nice improvement.